<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseWidget</class>
 <widget class="QWidget" name="ChooseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>请选择种族</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>双人</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>四人</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>请选择地图</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
